src/sentry/incidents/models.py

```diff
@@ -238,3 +244,64 @@ class Meta:
         app_label = 'sentry'
         db_table = 'sentry_incidentsuspectcommit'
         unique_together = (('incident', 'commit'), )
+
+
+class AlertRuleStatus(Enum):
+    PENDING = 0
+    TRIGGERED = 1
+    PENDING_DELETION = 2
+    DELETION_IN_PROGRESS = 3
+
+
+class AlertRuleThresholdType(Enum):
+    ABOVE = 0
+    BELOW = 1
+
+
+class AlertRuleAggregations(Enum):
+    TOTAL = 0
+    UNIQUE_USERS = 1
```

Comment:
Is this where you'll add in types like 75% and 95% ?

Reply:
Yeah, I think so. To some extent I'm not sure if this format is going to be too restrictive, but if we have a fixed list of percentiles it might work out ok. 

Alternative is that we could store the actual snuba aggregate query in the table, although one thing I like about these enums is that if the way we generate an aggregate changes then we just make a code change and don't need a backfill to update the aggregates.